server/integration/testsuite/src/test/resources/arquillian.xml

```diff
@@ -339,6 +339,20 @@
                 <property name="jmxDomain">${server.jmx.domain}</property>
             </configuration>
         </container>
+        <container qualifier="standalone-rest-ssl" mode="manual">
```

Comment:
Hmmm, do we really need a new container for this? Couldn't the hotrod-ssl container contain configuration to access rest over ssl? The more containers we have, the longer it takes to test :|

Reply:
Unfortunately no, this is a test for particular config template which we provide as part of ISPN distribution, i.e. each test in `ExampleConfigsIT` has to start its own container to test given template. We can either reduce number of templates we provide or skip testing of the templates (and especially later option I wouldn't recommend).